examples/playwright/src/theia-preference-view.ts

```diff
@@ -55,7 +56,11 @@ export const DefaultPreferences = {
     Explorer: {
         AutoReveal: {
             Enabled: true
+        },
+        DecorationsColors: {
+            Enabled: true
         }
+
```

Comment:
I believe changes in this file can be reverted, there are no `playwright` tests which test the functionality of the preference.